src/popup/components/Header.vue

```diff
@@ -26,7 +26,22 @@
       </Component>
     </div>
 
+    <BtnPlain
+      v-if="showActiveNetwork"
```

Comment:
I suggest to use the `showHeaderNavigation` instead of being dependant on the route name.

Reply:
But that button is only related to the current Active Network and it only navigates the networks page